qemu/tests/cfg/memhp_threads.cfg

```diff
@@ -0,0 +1,29 @@
+- memhp_threads:
+    type = memhp_threads
+    required_qemu = [5.0, )
+    paused_after_start_vm = yes
+    mem_fixed = 4096
+    slots_mem = 4
+    maxmem_mem = 32G
+    smp = 4
```

Comment:
I saw the smp = 4  but the scenaro is set to smp =1 ,we had better keep them the same.

Reply:
It's not necessary. It's about the increased threads number, not the original one.